Index: anytask/anytask/settings.py
===================================================================
--- anytask.orig/anytask/settings.py	2015-09-28 23:12:34.000000000 +0500
+++ anytask/anytask/settings.py	2015-09-28 23:15:20.950325604 +0500
@@ -40,6 +40,13 @@
 }

 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+MEDIA_ROOT = '/var/lib/anytask/media'
+UPLOAD_ROOT = '/var/lib/anytask/upload'
+STATIC_ROOT = '/var/lib/anytask/static'
+
+ # local overrides (optional)
+ import os
+

 # local overrides (optional)
 import os
